### ISSUE TITLE
Analyze percent of Gitcoin donors that used Safes

### ISSUE THREAD

**Issue by ccerv1** [hypercerts-org/hypercerts#451]

**Comment by ccerv1** [comment]
My recommendation would be to set the multisig's contract creator as the wallet that is able to make the claim. I've confirmed that person [has activity on Optimism](https://optimistic.etherscan.io/address/0xbd0531975d4d273e557e40856320304b39806ad8).

**Comment by holkeb** [comment]
Yes, works for me for now. Alternatively, we could also check with them, but your solution is ok. Only challenge is of course that it is not transferable, so it can't be changed later.

**Comment by holkeb** [comment]
OK, then your proposed solution is probably the best
